linux/switchtec.h

```diff
@@ -32,6 +32,7 @@
 #define SWITCHTEC_EVENT_EN_CLI   BIT(2)
 #define SWITCHTEC_EVENT_EN_IRQ   BIT(3)
 #define SWITCHTEC_EVENT_FATAL    BIT(4)
+#define SWITCHTEC_EVENT_NOT_SUPP BIT(31)
```

Comment:
This bit is not set anywhere and therefore not used. You should not merge commits that are not used by anything.

Reply:
I removed this patch and forced pushed the series. Will add this back when firmware is ready.